packages/scss/src/components/dialog/mods.scss

```diff
@@ -37,6 +37,7 @@
 	@include keyframe.slideFromBottom;
 
 	--components-dialog-animationOpening: slideFromBottom;
+	--components-dialog-maxHeight: calc(100vh - var(--spacings-S));
 	--components-dialog-maxHeight: calc(100dvh - var(--spacings-S));
```

Comment:
there is a double variable

Reply:
This is deliberate, so that browsers that don't understand the `dvh` unit have a fallback with the `vh` unit.

Reply:
That works on known properties, but do fallbacks work on custom variables?

Unless you define them as `@property` and the associated type checking, I would not expect them to do.